File: Api.Server/src/Services/Center/Config/Dto/EditConfigInput.cs
namespace Fast.Center.Service.Config.Dto;

/// <summary>
/// <see cref="EditConfigInput"/> 编辑配置输入
/// </summary>
public class EditConfigInput : UpdateVersionInput
{
    /// <summary>
    /// 配置Id
    /// </summary>
    [LongRequired(ErrorMessage = "配置Id不能为空")]
    public long ConfigId { get; set; }

    /// <summary>
    /// 配置名称
    /// </summary>
    [StringRequired(ErrorMessage = "配置名称不能为空")]
    public string ConfigName { get; set; }

    /// <summary>
    /// 配置值
    /// </summary>
    /// <remarks>
    /// <para>Boolean：[True, False]</para>
    /// </remarks>
    [StringRequired(ErrorMessage = "配置值不能为空")]
    public string ConfigValue { get; set; }

    /// <summary>
    /// 备注
    /// </summary>
    public string Remark { get; set; }
}

File: Web.Admin/src/api/services/Center/config/models/EditConfigInput.ts
/**
 * 编辑配置输入
 */
export interface EditConfigInput {
	/**
	 * 配置Id
	 */
	configId?: string;
	/**
	 * 配置名称
	 */
	configName?: string;
	/**
	 * 配置值
	 */
	configValue?: string;
	/**
	 * 备注
	 */
	remark?: string;
	/**
	 * 
	 */
	rowVersion?: string;
}

